docs/advanced_topics/accessibility_considerations.md

```diff
@@ -146,6 +146,69 @@ By default, the checker includes the following rules to find common accessibilit
 
 To customize how the checker is run (such as what rules to test), you can define a custom subclass of {class}`~wagtail.admin.userbar.AccessibilityItem` and override the attributes to your liking. Then, swap the instance of the default `AccessibilityItem` with an instance of your custom class via the [`construct_wagtail_userbar`](construct_wagtail_userbar) hook.
 
+For example, aXe's `p-as-heading` rule uses combinations of font weight and size or font weight plus italics to decide if a paragraph is acting as a heading visually. If your heading styles do not use italics, you may need aXe to rely solely on font weight to identify short, bold paragraphs as headings.
+
+```python
+from wagtail.admin.userbar import AccessibilityItem
+
+
+class CustomAccessibilityItem(AccessibilityItem):
+    axe_custom_checks = [
+        {
+	    # Flag heading-like paragraphs based only on font weight compared to surroundings.
+            "id": "p-as-heading",
+            "options": {
+                "margins": [
+                    { "weight": 150 },
+                ],
+                "passLength": 1,
+                "failLength": 0.5
+            },
+        },
+    ]
+
+
+@hooks.register('construct_wagtail_userbar')
+def replace_userbar_accessibility_item(request, items):
+    items[:] = [CustomAccessibilityItem() if isinstance(item, AccessibilityItem) else item for item in items]
+```
+
+The checks you run in production should be restricted to issues your content editors can fix themselves; warnings about things out of their control will only teach them to ignore all warnings. However, it may be useful for you to run additional checks in your development environment.
+
+```python
+from wagtail.admin.userbar import AccessibilityItem
+
+
+class CustomAccessibilityItem(AccessibilityItem):
+    # Run all AXE rules up to WCAG 2.2 AA
+    axe_rules_in_dev = [
+        "wcag2a",
+        "wcag2aa",
+        "wcag2aaa",
+        "wcag21a",
+        "wcag21aa",
+        "wcag22aa",
+        "best-practice",
+    ]
+    # Except for the color-contrast-enhanced rule
+    axe_rules = {
+        "color-contrast-enhanced": {"enabled": False},
+    }
+
+    def get_axe_run_only(self, request):
+        if env.bool('DEBUG', default=False):
+            return self.axe_rules_in_dev
+        else:
+            return self.axe_run_only  # Wagtail's default rules
```

Comment:
In production, run Wagtail's default accessibility rules for authored content only